### DISCUSSION TITLE
Add Information On How To turn off Parts of the Automod on the Automod Page

### DISCUSSION THREAD

**Discussion by NicholasK516** [jagrosh/Vortex#97] [Ideas]
I see a lot of people asking how to turn off the automod in the support server. There is no mention of how to do so on the [Automod Page](https://github.com/jagrosh/Vortex/wiki/Auto-Moderation). I feel like if this was on the automod page we could easily direct users to the page or they would find it themselves.

**Comment by jagrosh** [comment]
I've updated the [automod page](https://github.com/jagrosh/Vortex/wiki/Auto-Moderation) with the commands to disable each feature, which I agree is a good addition to the page. In a similar vein, I've also removed the automod setup command and default punishments, which I feel was leading to many users enabling features they didn't realize they were enabling.
